source/includes/steps-export.yaml

```diff
@@ -0,0 +1,70 @@
+title: Navigate to the :doc:`collection </collections>` you wish to export data into.
+level: 4
+ref: select-collection
+content: |
+  You can either select the collection from the
+  :doc:`Collections </collections>` tab or click the collection in the
+  left-hand pane.
+
+---
+title: Click :guilabel:`Collection` in the top-level menu and select :guilabel:`Export Collection`
+level: 4
+ref: export-collection
+content: |
+
+  .. raw:: html
+
+     <br>
+
+  .. figure:: /images/compass/export-data-option-select.png
+     :alt: Export data select menu
+
+  .. raw:: html
+
+     <br>
+
+  |compass-short| displays the following dialog:
+
+  .. raw:: html
+
+     <br>
+
+  .. figure:: /images/compass/export-data-dialog.png
+     :alt: Export data dialog
+
+  The top section of the export dialog displays the query
+  entered in the query bar prior to export, if applicable. You
+  can use this query to filter which collection documents are
+  exported. If no query was specified, this section displays
+  ``undefined``.
+
+---
+title: Use the :guilabel:`Export Full Collection` toggle to indicate 
+  whether to export only the documents matched by the query or the
+  full collection.
+level: 4
+ref: export-full-collection
+content: |
+  .. note::
+     Having an ``undefined`` query filter and toggling the
+     option to export the full collection both result in
+     |compass-short| exporting the full collection.
+
+---
+title: Choose the appropriate file type and the location of the export file.
```

Comment:
Thought: Maybe we split this into two steps since it's separate in the UI. We could maybe have one step for choosing the file type, and provide a context block that users can choose either csv or json. We could then have a second step for the export location.